ui/src/main/js/view/templates/run-changeset.hbs

```diff
@@ -1,9 +1,13 @@
 <div class="alert alert-info run-changeset-details {{kind}}-repo">
+    {{#if contributors.0.authorJenkinsId}}
     <div class="header">{{commitCount}} commit{{#ifCond commitCount '>' 1}}s{{/ifCond}} by <span class="contributor">{{contributors.0.authorJenkinsId}}</span>{{#ifCond contributorCount '>' 1}} and others{{/ifCond}}</div>
```

Comment:
:ant: Would seem easier and more accurate to do

```hbs
<div class="header">
  {{commitCount}} commit{{#ifCond commitCount '>' 1}}s{{/ifCond}}
  {{#ifCond contributors.0.authorJenkinsId}}
    by <span class="contributor">{{contributors.0.authorJenkinsId}}</span>
    {{#ifCond contributorCount '>' 1}} and others{{/ifCond}}
  {{/ifCond}}
</div>
```

(possibly with some XML whitespace manipulation there)